src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -437,6 +437,9 @@ public void finishLocalSetup() throws CheckstyleException {
         childContext = context;
     }
 
+    /**
+     * @noinspection ChainOfInstanceofChecks.
```

Comment:
Shouldn't this still start with `{@inheritDoc}` since it is an overridden method and you are not re-defining the description?